10-JavaScript/07-jquery-events.md

```diff
@@ -68,10 +69,78 @@ Hints:
 - Detecting keys like space and enter is a little different than letter keys. Look it up!
 - `console.log()` is your friend.
 
+## Advanced Event Handling
+### Binding to Multiple Elements
```

Comment:
I think we do want this for the keyboard exercise - since each key shouldn't have a separate event handler